src/Controller/Controller.js

```diff
@@ -0,0 +1,98 @@
+import Car from "../Model/Car.js";
+import InputView from "../View/InputView.js";
+import OutputView from "../View/OutputView.js";
+import AppError from "../utils/Error.js";
+
+const RESULT_MESSAGE = "실행 결과";
+const ERROR_MESSAGES = Object.freeze({
+  onlyNum: "숫자 값만 입력해주세요.",
+  invalidNumRange: "1 이상 200미만의 숫자만 입력해주세요.",
+  duplicateName: "중복된 이름이 있습니다.",
+});
```

Comment:
`freeze`를 사용해서 객체의 수정을 막는 것도 하나의 방법이지만 대문자를 사용해 상수임을 보다 명확하게 드러내는 것도 좋을거 같아요
```suggestion
const ERROR_MESSAGES = {
  ONLY_NUM: "숫자 값만 입력해주세요.",
  INVALID_NUM_RANGE: "1 이상 200미만의 숫자만 입력해주세요.",
  DUPLICATE_NAME: "중복된 이름이 있습니다.",
};
```

Reply:
`freeze`를 사용하게 되면 아래와 같은 코드가 있을 때 runtime에서 에러가 발생할텐데요.

상수에 대한 명확한 rule(ex - 상수는 모두 대문자 + `_`를 이용해 작성)을 가지면 이런 개발 단계에서 일어날 수 있는 실수를 막기 더 용이할거 같네용
``` 
ERROR_MESSAGES.onlynum = "some value"
```